communication/youtube/README.md

```diff
@@ -0,0 +1,25 @@
+# YouTube Management Project (The Stream Team)
+
+The Kubernetes community is an open community. As such, numerous meetings are
+recorded and shared through the
+[Kubernetes YouTube](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg)
+channel. Additionally, requests to live stream meetings are increasing as the
+community grows.
+
+## YouTube Channel Guidelines
+
+Please refer to [YouTube Channel Guidelines] with questions regarding YouTube.
+
+## Intent of Subproject
+
+This subproject's purpose is intended to be:
+
+1. Appropriately prioritize YouTube as a Kubernetes community resource and means
```

Comment:
Small nit on the list items 😬 As they are complete sentences, would you mind adding a period at the end?